### DISCUSSION TITLE
Any plans for internet access via wifi?

### DISCUSSION THREAD

**Discussion by mimimix** [zepp-health/zeppos-forum#87] [Q&A] [answered]
This will give the watch more independence from the phone.

**Comment by htoooth** [comment, marked as answer]
hi @Mimimix , 

We are considering whether to open this feature, mainly considering 
1. Power consumption 
3. Memory consumption. 
It is possible that this feature may not be opened.

**Reply by vaas1993** [reply]
我觉得在充分告知用户的前提下可以开放该功能，如果WiFi可以开启，或者说开启时选择开启时长的形式，应用程序就可以做更多的事情
I think this function can be opened on the premise that users are fully informed. If WiFi can be opened, or if the form of opening time is selected, the application can do more things
